cves/kernel/CVE-2017-18017.yml

```diff
@@ -452,7 +464,9 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    When coding in languages without garbage collection, it is important to make sure that resources are properly 
+    removed when you are finished with them.
```

Comment:
Seems a little cursory - how might a simple issue like this be prevented? What can we learn? Look at the broader context. Is the code that this happened in particularly complex? Unmaintained? Maintained by one person? Was this in the system for a long time? Show that you've gone deeper.

Reply:
Add more details, and explanations of what went wrong